src/main/java/uk/gov/hmcts/cmc/claimstore/jobs/cron/ClaimReissueJob.java

```diff
@@ -0,0 +1,35 @@
+package uk.gov.hmcts.cmc.claimstore.jobs.cron;
+
+import lombok.Getter;
+import org.quartz.DisallowConcurrentExecution;
+import org.quartz.JobExecutionContext;
+import org.quartz.JobExecutionException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.cmc.claimstore.services.ClaimReissueService;
+import uk.gov.hmcts.cmc.scheduler.model.CronJob;
+
+@Component
+@Getter
+@DisallowConcurrentExecution
+public class ClaimReissueJob implements CronJob {
+
+    private ClaimReissueService claimReissueService;
+    @Value("${claim_reissue.schedule}")
```

Comment:
value for this is needed in application.yml for it to run in local i think. Pls check.

Reply:
yes you are right. Added now.